packages/react-reconciler/src/ReactFiberHooks.js

```diff
@@ -1053,6 +1058,13 @@ function updateWorkInProgressHook(): Hook {
   return workInProgressHook;
 }
 
+function unstable_useContextWithBailout<T>(
+  context: ReactContext<T>,
+  compare: (T => mixed) | null,
```

Comment:
The name isn't super important b/c it's not going to be observed by anyone but I feel like `compare` is a confusing name for this argument because it doesn't do any comparison it just selects a value. why not call it `select` or something. I think you can land with compare just realized it was just chafing my automatic mental model while reviewing the PR

Reply:
Updated